tardis/model/base.py

```diff
@@ -0,0 +1,7 @@
+class Radial1DModel(object):
+    def __init__(self, plasma, atom_data, structure):
+        pass
+
+    @classmethod
+    def from_config(self, config):
```

Comment:
`velocities == np.hstack([v_inner, v_outer[:-1])`
It's something like that. So `velocities` has the same information as `v_inner` and `v_outer` combined.